### PR TITLE
Updated steps to deploy with lifecycle manager

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Updated steps to deploy with lifecycle manager

**Related issue(s)**

- https://github.com/kyma-project/nats-manager/issues/120

### CODE REVIEW

**Review by friedrichwilken — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -152,77 +152,35 @@ To delete the CRDs from the cluster:
...
-
-   It can be GitHub, DockerHub, GCP or a local registry.
-   If you haven't set up a container registry yet, refer to the following documentation:
-
-   - Lifecycle Manager: [Provision a cluster and OCI registry](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer-tutorials/provision-cluster-and-registry.md)
-  
-   - GitHub: [Working with the Container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). If you don't provide a registry Secret, change the visibility of a GitHub package to `public`.
-
-4. Generate the module template and push the container image. In the project root directory, run:
+   > **NOTE:** You can get the latest released module template from [here](https://github.com/kyma-project/nats-manager/releases/latest/download/module-template.yaml), or you can use module template from the artifacts of `nats-module-build` job from `main` branch or from your pull request.
```
Comment (NHingerl) [inline README.md:165]:
```suggestion
   > **NOTE:** You can get the latest released [module template](https://github.com/kyma-project/nats-manager/releases/latest/download/module-template.yaml), or you can use the module template from the artifacts of `nats-module-build` job either from the `main` branch or from your pull request.
```

File: README.md [hunk 2]
```diff
@@ -152,77 +152,35 @@ To delete the CRDs from the cluster:
...
-6. Apply the module template to the Kubernetes cluster:
+3. Enable the NATS module:
 
    ```sh
-   kubectl apply -f template.yaml
+   kyma alpha enable module nats -c fast -n kyma-system -p Ignore
    ```
 
-7. Deploy the `nats` module by adding it to `kyma` custom resource `spec.modules`:
+4. [Optional] Install NATS Custom Resource:
```
Comment (NHingerl) [inline README.md:177]:
```suggestion
4. Optionally, install NATS Custom Resource:
```

File: README.md [hunk 3]
```diff
@@ -152,77 +152,35 @@ To delete the CRDs from the cluster:
...
-     modules:
-     - name: nats
-   ...
-   ```
+    ```sh
+    kubectl apply -f config/samples/eventing-nats-eval.yaml
+    ```
 
-8. Check whether your NATS module is deployed properly:
+5. [Optional] Check whether your NATS module is deployed properly:
```
Comment (NHingerl) [inline README.md:183]:
```suggestion
5. If you want to verify whether your NATS module is deployed properly, perform the following checks:
```

**Review by NHingerl — COMMENTED** [review]
minor suggestions

**Review by NHingerl — APPROVED** [review]

**Pull request merged**